src/Docker.PowerShell/Cmdlets/ImportContainerImage.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Threading.Tasks;
+using System.Management.Automation;
+using Docker.DotNet.Models;
+using System.IO;
+using Newtonsoft.Json;
+using System.Text;
+using System.Linq;
+using Docker.PowerShell.Objects;
+
+namespace Docker.PowerShell.Cmdlets
+{
+    [Cmdlet(VerbsData.Import, "ContainerImage",
+            DefaultParameterSetName = CommonParameterSetNames.Default)]
+    [Alias("Load-ContainerImage")]
+    [OutputType(typeof(ImagesListResponse))]
+    public class ImportContainerImage : DkrCmdlet
+    {
+        private const string _loadedImage = "Loaded image: ";
```

Comment:
I know its tempting to think of constants as member vars but actually they should be `LoadedImage`